appeals/web/src/server/appeals/appeal-details/issue-decision/issue-decision.controller.js

```diff
@@ -460,42 +444,58 @@ export const renderLpaCostsDecisionLetterUpload = async (request, response) => {
 	});
 };
 
+/**
+ * @param {object} params
+ * @param {import('got').Got} params.apiClient
+ * @param {{appealId: number|string, folderId: number, letterDate: string, files: { GUID: string, name: string, documentType: string, size: number, stage: string, mimeType: string, receivedDate: string, redactionStatus: number, blobStoreUrl: string }[] }} params.decision - The decision object.
```

Comment:
would it be worth pulling out this type definition and defining it separately so it can be named, and more easily reused?